portfolio/src/main/webapp/script.js

```diff
@@ -32,16 +32,34 @@ function funFactGenerator() {
   factGetter.innerText = fact;
 }
 
-function serverFunFact() {
-    
+/**
+ * Displays a fun fact about Desmond on the page.
+ */
+function serverFunFact() { 
     fetch('/data').then(facts => facts.json()).then(jsonfacts => {
 
-    // Pick a random fact.
-    const fact = jsonfacts[Math.floor(Math.random() * jsonfacts.length)];
+        // Pick a random fact.
+        const fact = jsonfacts[Math.floor(Math.random() * jsonfacts.length)];
+
+        // Add it to the page.
+        const factGetter = document.getElementById('factgetter');
+        factGetter.innerText = fact;
+    })
+}
+
+/**
+ * Adds user's comment to the page.
```

Comment:
is this description still valid? You may want to explain that it retrieves all comments as well?

Reply:
For now this description is still valid... it only adds the last comment indexed of the comments arraylist (which is the one the user just posted) *line 60/61*... it wont be valid once I implement the datastore though so I'll be sure to update that comment when I implement that.